base/operators.jl

```diff
@@ -851,10 +852,14 @@ julia> fs = [
 
 julia> ∘(fs...)(3)
 3.0
+
+julia> ∘(fs...) === compose(fs...)
```

Comment:
I admit to be surprised by this property: is it documented or an implementation detail of functions returning anonymous functions?

Reply:
For `union` we don't even bother to use the unicode equivalent in the docstring. I think just repeating the previous example but with `compose` would be enough (`compose(fs...)(3) ==> 3.0`).

Reply:
@rfourquet Thanks for the suggestion. It makes sense. I updated the docstring.